pallets/mnt-token/src/tests.rs

```diff
@@ -2,16 +2,601 @@
 
 use super::Error;
 use crate::mock::*;
-
 use frame_support::{assert_noop, assert_ok};
-use minterest_primitives::{CurrencyId, Rate};
+use minterest_primitives::{Balance, CurrencyId, Rate};
+use orml_traits::MultiCurrency;
 use sp_arithmetic::FixedPointNumber;
+use sp_runtime::traits::Zero;
 
 const KSM: CurrencyId = CurrencyId::KSM;
 const DOT: CurrencyId = CurrencyId::DOT;
 const ETH: CurrencyId = CurrencyId::ETH;
 const BTC: CurrencyId = CurrencyId::BTC;
 
+const MNT_PALLET_STARED_BALANCE: Balance = 1000000 * DOLLARS;
```

Comment:
```suggestion
const MNT_PALLET_START_BALANCE: Balance = 1000000 * DOLLARS;
```
Is that what you mean?